From david-b@pacbell.net Tue Apr 17 13:09:33 2007
From: Raphael Assenat <raph@8d.com>
Date: Tue, 17 Apr 2007 13:09:18 -0700
Subject: USB: quirk for broken suspend of IT8152F/G
To: Greg KH <greg@kroah.com>
Cc: Raphael Assenat <raph@8d.com>
Message-ID: <200704171309.19008.david-b@pacbell.net>


From: Raphael Assenat <raph@8d.com>

Here's a patch which adds my device to the list.

This patch enables the broken suspend quirk for the PCI OHCI controller
present in the IT8152F/G RISC-to-PCI Companion Chip.

Signed-off-by: Raphael Assenat <raph@8d.com>
Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ohci-pci.c |    4 ++++
 1 file changed, 4 insertions(+)

--- a/drivers/usb/host/ohci-pci.c
+++ b/drivers/usb/host/ohci-pci.c
@@ -141,6 +141,10 @@ static const struct pci_device_id ohci_p
 		.subdevice	= 0x0004,
 		.driver_data	= (unsigned long) broken_suspend,
 	},
+	{
+		PCI_DEVICE(PCI_VENDOR_ID_ITE, 0x8152),
+		.driver_data = (unsigned long) broken_suspend,
+	},
 	/* FIXME for some of the early AMD 760 southbridges, OHCI
 	 * won't work at all.  blacklist them.
 	 */
